raiden/network/upnpsock.py

```diff
@@ -8,6 +9,17 @@
 log = slogging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def valid_ip_v4(address):
+    try:
+        ipaddress.ip_address(address)
+    except ipaddress.AddressValueError:
+        return valid_ip_v4(unicode(address))
```

Comment:
Nitpick: We should probably use `unicode(address, errors='ignore')` to avoid `UnicodeDecodeError`s in case the input contains something else than an IP-Address.